README.md

```diff
@@ -284,6 +163,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job so far @henokkhm you've done a great job with this project and your README is well setup well done 💯 . There are just a few changes you need to make and you'll be on your way to the next project 😉 
- [ ] I noticed that your MIT License currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink:
- [ ] You might be wondering what `The MIT License ` is and why it is important and this is because it is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. 💯 

<img width="1280" alt="Screen Shot 2023-04-25 at 8 25 20 PM" src="https://user-images.githubusercontent.com/102699958/234395628-71d66c7b-4842-4722-a0f4-22ace0d01d1c.png">

Reply:
@NahnahAJ  Thank you for your feedback. I have made all the changes you suggested.